examples/ensemble/plot_gradient_boosting_categorical.py

```diff
@@ -138,26 +138,17 @@
 # -----------------------------------------------------------
 # We now create a :class:`~ensemble.HistGradientBoostingRegressor` estimator
 # that will natively handle categorical features. This estimator will not treat
-# categorical features as ordered quantities.
-#
-# Since the :class:`~ensemble.HistGradientBoostingRegressor` requires category
-# values to be encoded in `[0, n_unique_categories - 1]`, we still rely on an
-# :class:`~preprocessing.OrdinalEncoder` to pre-process the data.
-#
-# The main difference between this pipeline and the previous one is that in
+# categorical features as ordered quantities. We set
+# `categorical_features="by_dtype"` such that features with categorical dtype
+# are considered categorical features.
+
```

Comment:
```suggestion
#
```

Otherwise the formatting is broken:

![image](https://github.com/scikit-learn/scikit-learn/assets/89061/1d539eb5-90a9-4501-a429-8739826a5708)